server/src/main/java/org/eclipse/lsp/cobol/core/preprocessor/delegates/util/RangeUtils.java

```diff
@@ -114,17 +70,16 @@ public boolean isBefore(Position after, Position before) {
     return compareTo(after, before) < 0;
   }
 
-  /**
-   * Tests if text document position resides inside the location.
-   *
-   * @param uri the uri of the node
-   * @param position the position of the node to add
-   * @param location the locality of the wrapping node
-   * @return true if position is inside the location.
-   */
-  public boolean isInside(String uri, Position position, Locality location) {
+  private boolean isInside(String uri, Position position, Locality location) {
```

Comment:
You can keep the docs even for private functions.

Reply:
Makes sense.